pkg/git/git.go

```diff
@@ -1,12 +1,13 @@
 package git
 
 import (
+	"time"
```

Comment:
Why is that re-formatted?

Reply:
yepp, my visual studio automaticall reformats on save. the reformattiing make sense from MPOV.

Reply:
> the reformattiing make sense from MPOV.

But the normative formatting in this project so far is `go fmt ./...`. This format is also asserted [here](https://github.com/SAP/jenkins-library/blob/master/.github/workflows/verify-go.yml#L53), which is used by some automated voter.